packages/@angular/cli/lib/config/schema.json

```diff
@@ -105,6 +105,10 @@
             "type": "string",
             "description": "The name of the TypeScript configuration file for unit tests."
           },
+          "karma": {
```

Comment:
shouldn't be a breaking change as the string form of `test` would still function.  Similar to the scripts options.